client/web/compose/src/components/ModuleFields/Editor/Geometry.vue

```diff
@@ -180,51 +180,32 @@
         </div>
       </div>
 
-      <l-map
-        ref="map"
-        :zoom="map.zoom"
-        :center="map.center"
-        style="height: 75vh; width: 100%; cursor: pointer;"
-        @click="placeMarker"
-        @locationfound="onLocationFound"
-      >
-        <l-tile-layer
-          url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
-          :attribution="map.attribution"
-        />
-        <l-marker
-          v-for="(marker, i) in markers"
-          :key="i"
-          :lat-lng="marker"
-          :opacity="localValueIndex === undefined || i == localValueIndex ? 1.0 : 0.6"
-          @click="removeMarker(i)"
-        />
-        <l-control class="leaflet-bar">
-          <a
-            v-if="!field.options.hideCurrentLocationButton"
-            :title="$t('tooltip.goToCurrentLocation')"
-            role="button"
-            class="d-flex justify-content-center align-items-center"
-            @click="goToCurrentLocation"
-          >
-            <font-awesome-icon
-              :icon="['fas', 'location-arrow']"
-              class="text-primary"
-            />
-          </a>
-        </l-control>
-      </l-map>
+      <c-map
+        ref="c-map"
+        :field="field"
+        :map="map"
+        :on-location-found="onLocationFound"
+        :hide-current-location-button="!field.options.hideCurrentLocationButton"
+        :local-value-index="localValueIndex"
+        :markers="markers"
+        :label="{
+          tooltip: { 'goToCurrentLocation': $t('tooltip.goToCurrentLocation') }
+        }"
+        @on-go-to-current-location="goToCurrentLocation"
+        @on-place-marker="placeMarker($event)"
```

Comment:
this should be on-map-click="placeMarker"
Also you dont need the $event if you don't use it with (), so `@on-map-click="placeMarker""